experimental/dsh/dsh.c

```diff
@@ -1767,13 +1768,14 @@ int main(int argc, char** argv)
         {"lite",     0, 0, 'l'},
         {"help",     0, 0, 'h'},
         {"verbose",  0, 0, 'v'},
+        {"text",     0, 0, 't'},
         {0, 0, 0, 0}
     };
 
     int usage = 0;
     while (1) {
         int c = getopt_long(
-                    argc, argv, "f:i:o:lhv",
+                    argc, argv, "f:i:o:lhvrt",
```

Comment:
Thanks @KnightKu.  Looks good, but let's drop the "r" here.  At least I think you mean to drop it.

Reply:
yeah, true, forgot it here, let me fix it.